### DISCUSSION TITLE
Scala "Maintained" Status?

### DISCUSSION THREAD

**Discussion by ryaminal** [nvim-treesitter/nvim-treesitter#1907] [Q&A]
Hi all,

What is the status on developing the Scala implementation to be "maintained"(and therefore install-able with `ensure_installed="maintained"` ?

I haven't been able to track down anything in this repo or the tree-sitter-scala repo that indicates why it's not "maintained" currently. It appears that it's actively being worked on so I'm assuming there is some other requirement.

**Comment by theHamsta** [comment]
Maintained does not refer to the parser but someone caring that the queries. Create a pull request and add your name here if you want to take care of the scale queries in this repo. It will automatically be `maintained`

https://github.com/theHamsta/nvim-treesitter/blob/70b1997cd7605d7be5c1cea86019281cdc296386/lua/nvim-treesitter/parsers.lua

**Reply by theHamsta** [reply]
Now scala is maintained: https://github.com/nvim-treesitter/nvim-treesitter/pull/1909
